bedrock/security/models.py

```diff
@@ -67,7 +67,7 @@ def save(self, force_insert=False, force_update=False,
 class SecurityAdvisory(models.Model):
     id = models.CharField(max_length=8, primary_key=True, db_index=True)
     title = models.CharField(max_length=200)
-    impact = models.CharField(max_length=100)
+    impact = models.CharField(max_length=100, null=True)
```

Comment:
And Django defaults to `""` for CharFields according to https://github.com/django/django/blob/master/django/db/models/fields/__init__.py#L774